config/translations/en/module.download.yml

```diff
@@ -3,6 +3,9 @@ arguments:
     module: 'Module or modules to be enabled should be separated by a space'
 options:
     latest: 'Default to download most recent version'
+    module-path: 'The path of the contrib project'
```

Comment:
Can you change all module-path to just path ? We already are inside of the module it seem redundant.